src/characteristics/amperes.ts

```diff
@@ -1,15 +1,14 @@
-import type { Characteristic, WithUUID } from 'homebridge';
+import type { WithUUID } from 'homebridge';
+import DefaultCharacteristicClass from './types';
 
 export default function amperes(
-  DefaultCharacteristic: typeof Characteristic
-): WithUUID<new () => Characteristic> {
+  DefaultCharacteristic: typeof DefaultCharacteristicClass
+): WithUUID<new () => DefaultCharacteristicClass> {
   return class Amperes extends DefaultCharacteristic {
     static readonly UUID = 'E863F126-079E-48FF-8F27-9C2605A29F52';
 
     constructor() {
-      // @ts-ignore: unable to override class constructor parameters as its a type and not a value
       super('Amperes', Amperes.UUID, {
-        // @ts-ignore: custom unit
```

Comment:
Created `DefaultCharacteristicClass` to be able to get rid of 2 `// @ts-ignore` comments for every characteristics